bundles/org.openhab.binding.xmltv/src/main/java/org/openhab/binding/xmltv/internal/handler/XmlTVHandler.java

```diff
@@ -67,12 +69,11 @@ public void initialize() {
         logger.debug("Initializing {} for input file '{}'", getClass(), config.filePath);
 
         reloadJob = scheduler.scheduleWithFixedDelay(() -> {
-            final StreamSource source = new StreamSource(config.filePath);
             currentXmlFile = null;
             XMLStreamReader xsr = null;
             try {
                 // This can take some seconds depending upon weight of the XmlTV source file
-                xsr = xif.createXMLStreamReader(source);
+                xsr = xif.createXMLStreamReader(new FileInputStream(new File(config.filePath)), config.encoding);
```

Comment:
Is it really needed to have the encoding here? Using `FileInputStream` should return binary data and in my understanding this should be workling out of the box.

Reply:
Unfortunately not and I don't know why.
Tried it without encoding, for example this text should be: "beszéd" and I got "beszĂŠd".